docs/ref/contrib/staticfiles.txt

```diff
@@ -285,11 +285,17 @@ following requirements are met:
 * the :setting:`STATICFILES_STORAGE` setting is set to
   ``'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'``
 * the :setting:`DEBUG` setting is set to ``False``
-* you use the ``staticfiles`` :ttag:`static<staticfiles-static>` template
-  tag to refer to your static files in your templates
 * you've collected all your static files by using the
   :djadmin:`collectstatic` management command
 
+.. versionchanged:: 1.10
+
+    * you no longer have to use ``staticfiles`` :ttag:`static<staticfiles-static>`
```

Comment:
You can drop the bullet and start it like this: "In older versions, you also had to use the..."